src/Controller/Async/FilesystemManager.php

```diff
@@ -88,12 +88,15 @@ public function browse(Request $request, $namespace, $path)
 
         $files = $filesystem->find()->in($path)->files()->depth(0)->toArray();
         $directories = $filesystem->find()->in($path)->directories()->depth(0)->toArray();
+        
+        $multiselect = $request->get('multiselect');
 
         $context = [
             'namespace'    => $namespace,
             'files'        => $files,
             'directories'  => $directories,
             'pathsegments' => $pathsegments,
+            'multiselect'  => true,
         ];
```

Comment:
You probably just want:
```php
        $context = [
            'namespace'    => $namespace,
            'files'        => $files,
            'directories'  => $directories,
            'pathsegments' => $pathsegments,
            'multiselect'  => $request->query->get('multiselect'),
        ];
```

But no parameters are being passed into the request currently, so that's why you need to hardcode.